[PP-3952] Update cohort import documents (#7980)
braze-inc/braze-docs: _docs/_partners/data_and_infrastructure_agility/cohort_import/amplitude.md

![In the Braze segment builder, the filter "amplitude_cohorts" is set to "includes_value" and "Amplitude cohort test".]({% image_buster /assets/img/amplitude2.png %})

After saving, you can reference this segment during Canvas or campaign creation in the targeting users step.

## User Matching

> Identified users can be matched on either `external_ids` or `aliases`. Anonymous users can be matched on  `device_ids`. Identified users previously created as anonymous users with device_id can not be matched with device ids. You must use external IDs or aliases for identified users.

Reviewer: ```suggestion
Identified users can be matched by either their `external_id` or `alias`. Anonymous users can be matched by their `device_id`.
```

Could keep it simple by just stating what is possible.

> Identified users can be matched on either `external_ids` or `aliases`. Anonymous users can be matched on  `device_ids`. Identified users previously created as anonymous users with device_id can not be matched with device ids. You must use external IDs or aliases for identified users.

Author: True, however the scenario we are seeing alot is that customers are asking why users that were able to be identified by device_id are no longer being matched by device_id after they've been identified. So I wonder if there is a way to phrase that in a simple way.

@rachel-feinberg fyi

> Identified users can be matched on either `external_ids` or `aliases`. Anonymous users can be matched on  `device_ids`. Identified users previously created as anonymous users with device_id can not be matched with device ids. You must use external IDs or aliases for identified users.

Reviewer: Identified users can be matched by either their `external_id` or `alias`. Anonymous users can be matched by their `device_id`. Identified users who were originally created as anonymous users can't be identified by their `device_id`, and must be identified by their `external_id` or `alias`.

How does that sound?

> Identified users can be matched on either `external_ids` or `aliases`. Anonymous users can be matched on  `device_ids`. Identified users previously created as anonymous users with device_id can not be matched with device ids. You must use external IDs or aliases for identified users.

Author: sounds good to me
